TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyInterface.php

```diff
@@ -17,12 +17,4 @@
  */
 interface ProxyInterface
 {
-    /**
-     * Wake up method.
-     *
-     * Proxies need to have one as at least session handling relies on it.
-     *
-     * @return void
-     */
-    public function __wakeup();
```

Comment:
@kdambekalns if you look at what the code actually did before the change was to check for related entites and reconstitute them. With the change we know beforehand if there are related entities so we can skip the __wakeup method. I haven'T found a single place where we explicitly call __wakeup without checking if it actually exists...
If you insist I could keep it in the interface and create an empty wakeup in the proxy classes....